src/commands/addBinding/settingSteps/BooleanPromptStep.ts

```diff
@@ -9,10 +9,15 @@ import { BindingSettingStepBase } from "./BindingSettingStepBase";
 
 export class BooleanPromptStep extends BindingSettingStepBase {
     public async promptCore(context: IBindingWizardContext): Promise<string> {
-        const picks: QuickPickItem[] = [
+        let picks: QuickPickItem[] = [
             { label: 'true', description: '' },
             { label: 'false', description: '' }
         ];
-        return (await context.ui.showQuickPick(picks, { placeHolder: this._setting.label })).label;
+
+        if (this._setting.defaultValue?.toLowerCase() === 'false') {
+            picks = picks.reverse();
+        }
```

Comment:
Could you elaborate on why you reverse the picks here? Also why add `this._setting.description` as a potential placeholder below?

Reply:
> Could you elaborate on why you reverse the picks here?

If the default is "false", I just want "false" to be first.

> Also why add this._setting.description as a potential placeholder below?

So "label" is generally the name of the setting and "description" is a description of it. For input boxes I can use both ("label" for "placeHolder" and "description" for "prompt"), but for quick picks I only have "placeHolder". We hardly ever use BooleanPromptStep, and it just happened to look better in this case which is why I added it. The only risk is that placeHolder can get cut off if it's too long (a description will generally be longer than a label), but you can still hover over it to see the tooltip, which has the whole thing.